.fixtures.yml

```diff
@@ -27,7 +27,6 @@ fixtures:
       ref: "1.9.0"
     augeas: "camptocamp/augeas"
     cron: "puppet/cron"
-    augeas_core: "puppetlabs/augeas_core"
```

Comment:
This is needed for spec tests to pass with Puppet6.

Reply:
thanks, added.  Do we want to be testing against P6 as this module (and probably many deps) show >6 ?

Reply:
I personally am in favor of supporting Puppet6 (thus updating the metadata), but only after gaining some confidence (thus the spec tests).